demos/bluetooth/mesh/thingy52/common/model/thingy52.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file
+ *  @defgroup bt_mesh_thingy52 Thingy52 models
+ *  @{
+ *  @brief API for the Thingy52 models.
+ */
+
+#ifndef BT_MESH_THINGY52_H__
+#define BT_MESH_THINGY52_H__
+
+#include <bluetooth/mesh/model_types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** RGB colors */
+struct bt_mesh_thingy52_rgb {
+	/** Color Red */
+	uint8_t red;
+	/** Color Green */
+	uint8_t green;
+	/** Color Blue */
+	uint8_t blue;
+};
+
+/** Thingy52 RGB message */
+struct bt_mesh_thingy52_rgb_msg {
+	/** Remaining hops of the message */
+	uint16_t ttl;
+	/** Delay assosiated with the message */
+	uint16_t delay;
+	/** Color of the message. */
+	struct bt_mesh_thingy52_rgb color;
+	/** Speaker status of the message. */
+	bool speaker_on;
+};
+
+/** @cond INTERNAL_HIDDEN */
+
+#define BT_MESH_NORDIC_SEMI_COMPANY_ID 0x0059
+#define BT_MESH_MODEL_ID_THINGY52_SRV 0x0005
+#define BT_MESH_MODEL_ID_THINGY52_CLI 0x0006
+
+#define BT_MESH_THINGY52_OP_RGB_SET                                            \
+	BT_MESH_MODEL_OP_3(0x83, BT_MESH_NORDIC_SEMI_COMPANY_ID)
```

Comment:
Opcode `0xC3` is already reserved by `Simple OnOff Set Unreliable` message: https://infocenter.nordicsemi.com/topic/com.nordic.infocenter.meshsdk.v5.0.0/simple__on__off__common_8h_source.html

Could you use another message opcode? All after `0xC8` are free.

Reply:
Btw, you don't need to set the 2-most significant bits in the opcode to 1. `BT_MESH_MODEL_OP_3` will do that for you.

Reply:
I've assigned opcode `0x09` for `Thingy52 RGB Set` message in this PR: https://github.com/nrfconnect/sdk-nrf/pull/3549 You can suggest other opcode if you want.